src/index.html

```diff
@@ -1,14 +1,367 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Human rights</title>
-  <link rel="stylesheet" href="style.css">
-</head>
-<body>
-  <h1>Human rights</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Human rights</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1 class="udohr"> Universal Declaration of Human Rights </h1>
+    <h2 class="lol"> Preamble </h2>
+    <p class="kek">
+      Whereas recognition of the inherent dignity and of the equal and inalienable
+      rights of all members of the human family is the foundation of freedom, justice
+      and peace in the world,
+      <br>
+        Whereas disregard and contempt for human rights have resulted in barbarous
```

Comment:
```suggestion
      <br>
      Whereas disregard and contempt for human rights have resulted in barbarous
```
This text is not inside of br element, so there's no need to make another indent